Source/gmenu.cpp

```diff
@@ -174,7 +174,7 @@ void gmenu_draw()
 
 void gmenu_draw_menu_item(TMenuItem *pItem, int y)
 {
-	DWORD x, w, nSteps, step, pos, t;
+	DWORD w, x, nSteps, step, pos, t;

```

Comment:
Did the order of these variables determine binary exactness in Hellfire? If so, cool. Didn't know order of locals affected the asm output.

Reply:
Indeed: https://github.com/diasurgical/devilution/pull/1670

Reply:
Typically it only determines the order used in `lea` instructions. For example, we had lots of `lea ecx,[ebp+edx]` vs `lea ecx,[edx+eb]` fixed by this. VC6 does a better job at optimizing, so it didnt matter there.